src/components/businessUnitSplit/buSplit.js

```diff
@@ -378,17 +375,61 @@ function BusinessUnitSplit() {
       if (selectedFile.file) {
         let reader = new FileReader();
         reader.onload = (e) => {
-          console.log("reader onload");
           let result = e.target.result;
           let workbook = xlsx.read(result, { type: "array" });
           let sheetName = workbook.SheetNames[0];
           let worksheet = workbook.Sheets[sheetName];
           let json = xlsx.utils.sheet_to_json(worksheet);
           let errorJson = [];
-          console.log("Reading excel: ", json);
+          json.forEach((i) => {
+            if (i.SP) {
+              if (isNaN(i.SP)) {
+                errorJson.push(
+                  `Bu split accepts only Numeric value in - ${i.Partner_Account_Name} partner`
```

Comment:
SP field accepts only Numeric value for partner - <partner name>

same applies for all the fields